packages/fw-theme-builder/src/fw-theme-builder.ts

```diff
@@ -0,0 +1,325 @@
+import { html, css, LitElement } from "lit";
+import { customElement, property, state } from "lit/decorators.js";
+import "./fw-color-pick";
+import "./fw-size-pick";
+import "./fw-font-pick";
+import { fontOptions, defaultTheme } from "./models";
+
+enum ThemeEnum {
+  COLORS = "colors",
+  FONTS = "fonts",
+  SIZES = "sizes"
+}
+
+type Theme = {
+  colors: {[key: string] : any},
+  fonts: {[key: string] : any},
+  sizes: {[key: string] : any},
+}
+
+@customElement("fw-theme-builder")
+export class FwThemeBuilder extends LitElement {
+  @state()
+  nav = "Home";
+
+  @property({ type: Array })
+  fontOptions = fontOptions;
+
+  @property({ type: Object })
+  theme : Theme = defaultTheme;
+
+  @property({ type: Boolean })
+  viewByGroup = false;
+
+  static styles = css`
+    .back-button {
+      user-select: none;
+      cursor: pointer;
+      width: 3rem;
+      padding: 0.5rem 0.5rem;
+      z-index: 100;
+    }
+    .back-hidden {
+      opacity: 0;
+      cursor: default;
+    }
+    .back-icon {
+      height: 1rem;
+    }
+
+    .action-button {
+      transition: all 0.1s ease-in-out;
+      cursor: pointer;
+      width: 3.5rem;
+      user-select: none;
+      text-align: center;
+      border-radius: 4px;
+      font-family: "DM Sans", sans-serif;
+      padding: 0.5rem 0.5rem;
+      box-shadow: #1b1b1b3b 0px 4px 2px;
+    }
+    .save-btn {
+      background-color: #aaf16f;
```

Comment:
Do we use these?

If so, use css variables.

Can also verify if there's any redundant css or methods.